src/components/atom/TextInput/types.ts

```diff
@@ -4,7 +4,7 @@ export type TextInputProps = {
     placeholder?: string;
     value?: string;
     onChange?: () => void;
-    onPressEnter?: () => void;
+    onPressEnter?: (e: any) => void;
```

Comment:
Can you check the original type for `onPressEnter`? We should try to avoid using `any` as much as possible.